test/supertest.js

```diff
@@ -815,6 +815,17 @@ describe('request.agent(app)', function() {
   });
 });
 
+describe('request.agent(app, {prefix})', function() {
+  it('should apply prefix', function(done) {
```

Comment:
could you provide a successful test (e.g. with 200 status code) as well?

Reply:
Oh yeah, I can't believe I didn't do that already!